src/directives/class-map.ts

```diff
@@ -23,20 +22,21 @@ export interface ClassInfo {
  * Stores the ClassInfo object applied to a given AttributePart.
  * Used to unset existing values when a new ClassInfo object is applied.
  */
-const previousClassesCache = new WeakMap<Part, Set<string>>();
+const previousClassesCache = new WeakMap<Part, Map<string, unknown>>();
```

Comment:
I think this deserves a comment explaining what's happening, since the key in the Map is now either a class name or a static string...

Reply:
Done.